charts/cpo-case-payment-orders-api/values.yaml

```diff
@@ -30,4 +30,6 @@ java:
     S2S_AUTHORIZATIONS_XUI_ID: xui_webapp
     S2S_AUTHORIZATIONS_XUI_PERMISSION: R
     S2S_AUTHORIZATIONS_PAYMENT_ID: payment_app
-    S2S_AUTHORIZATIONS_PAYMENT_PERMISSION: R
\ No newline at end of file
+    S2S_AUTHORIZATIONS_PAYMENT_PERMISSION: R
+    S2S_AUTHORIZATIONS_FPRL_ID: fprl-cos-api
```

Comment:
we might need to add fprl into normal s2s whitelist as well?

Reply:
Forgot how CPO worked 😄 Done